meteor/client/ui/Settings/ConfigManifestSettings.tsx

```diff
@@ -188,21 +190,21 @@ function getEditAttribute<DBInterface extends { _id: ProtectedString<any> }>(
 			if (item.multiple) {
 				return renderMultiSelect(attribute, object, selectOptions, collection)
 			}
-			return renderDropdown(attribute, object, selectOptions, collection)
+			return renderDropdown(attribute, object, selectOptions, collection, undefined, allowNoSelection)
 		}
 		case ConfigManifestEntryType.SOURCE_LAYERS: {
 			const filterSourceLayerOptions = 'options' in item ? item.options : filterSourceLayers(item, sourceLayers ?? [])
 			if (item.multiple) {
 				return renderMultiSelect(attribute, object, filterSourceLayerOptions, collection)
 			}
-			return renderDropdown(attribute, object, filterSourceLayerOptions, collection)
+			return renderDropdown(attribute, object, filterSourceLayerOptions, collection, undefined, allowNoSelection)
 		}
 		case ConfigManifestEntryType.LAYER_MAPPINGS: {
 			const layerMappingOptions = 'options' in item ? item.options : filterLayerMappings(item, layerMappings ?? {})
 			if (item.multiple) {
 				return renderMultiSelect(attribute, object, layerMappingOptions, collection)
 			}
-			return renderDropdown(attribute, object, layerMappingOptions, collection)
+			return renderDropdown(attribute, object, layerMappingOptions, collection, undefined, allowNoSelection)
```

Comment:
What's the point of providing the parameter `undefined` here?

Reply:
We reached a point when this function has so many arguments that it makes no sense anymore. Luckily React components have props that are named for good readability and can be omitted when undefined, how about we use that?